application.example.yaml

```diff
@@ -15,6 +15,13 @@ notifications:
     # exchange's API is. Because there are so many factors it made sense to be able to configure the warning so you can
     # control how noisy your logs are.
     slowTickerWarning: 4000
+  email:
```

Comment:
It's `email` here and `mail` in the other `application.yaml`. Ideally I'd like all the SMTP configuration to be here, but if it can't I'd like them to at least agree with one another.